packages/web3-rpc-providers/CHANGELOG.md

```diff
@@ -37,10 +37,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.0.0.rc.0]
 
-#### Added
+### Added
 
 -   RC release 
 
-## [Unreleased]
+## [1.1.0.rc.0]
```

Comment:
This should be 1.0.0-rc.1 as we can do multiple RCs with breaking changes until 1.0.0 is stable or final to release.